rllib/agents/slateq/slateq_torch_policy.py

```diff
@@ -87,14 +91,18 @@ def build_slateq_losses(
 
     # Q-value computations.
     # ---------------------
-    observation = train_batch[SampleBatch.OBS]
+    # action.shape: [B, S]
+    actions = train_batch[SampleBatch.ACTIONS]
+
+    observation = convert_to_torch_tensor(
```

Comment:
just a thought, should we fix this in a more generic way for dict obs space?
not sure if this is also happening to other agents, and they just don't use strange looking obs by default.